elements/rh-audio-player/test/rh-audio-player.spec.ts

```diff
@@ -22,499 +25,423 @@ describe('<rh-audio-player>', async function() {
+      ui.mute = getButton('#mute');
+      ui.time = getShadowElementBySelector('#time');
+      ui.poster = getButton('#poster');
+      ui.volume = getShadowElementBySelector('#volume');
+      ui.forward = getButton('#forward');
+      ui.rewind = getButton('#rewind');
+      ui.playbackrate = getShadowElementBySelector('#playback-rate');
+      ui.playbackratestepdown = getButton('#playback-rate-stepdown');
+      ui.playbackratestepup = getButton('#playback-rate-stepup');
+      ui.fullplaybackrate = getShadowElementBySelector('#full-playback-rate');
+      ui.fullplaybackratestepdown = getButton('#full-playback-rate-stepdown');
+      ui.fullplaybackratestepup = getButton('#full-playback-rate-stepup');
+      ui.menu = getButton('#menu');
+    };
+
+    before(async function() {
+      // TODO: there is an issue with media seeking on Chrome that affects local files, similar to this issues: https://stackoverflow.com/questions/8088364/html5-video-will-not-loop
       element = await createFixture <RhAudioPlayer>(html`
         <rh-audio-player poster="https://www.redhat.com/cms/managed-files/CLH-S7-ep1.png">
           <p slot="series">Code Comments</p>
           <h3 slot="title">Bringing Deep Learning to Enterprise Applications</h3>
-          <rh-audio-player-about slot="about">
-            <h4 slot="heading">About the Episode</h4>
-            <p>
-              There are a lot of publicly available data sets out there. But when it
-              comes to specific enterprise use cases, you&apos;re not necessarily going to
-              able to find one to train your models. To realize the power of AI/ML in
-              enterprise environments, end users need an inference engine to run on
-              their hardware. Ryan Loney takes us through OpenVINO and Anomalib, open
-              toolkits from Intel that do precisely that. He looks specifically at
-              anomaly detection in use cases as varied as medical imaging and
-              manufacturing.
-            </p>
-            <p>
-              Want to learn more about Anomalib? Check out the research paper that
-              introduces the deep learning library.
-            </p>
-            <rh-audio-player-profile slot="profile" src="https://www.redhat.com/cms/managed-files/ryan-loney.png">
-              <span slot="fullname">Ryan Loney</span><br>
-              <span slot="role">Product manager, OpenVINO Developer Tools</span>, <span slot="company">Intel&reg;</span>
-            </rh-audio-player-profile>
-          </rh-audio-player-about>
           <audio crossorigin="anonymous" slot="media" controls>
-            <source type="audio/mp3" srclang="en" src="../demo/detailed-transcript.mp3">
+            <source type="audio/mp3" srclang="en" src="https://cdn.simplecast.com/audio/28d037d3-7d17-42d4-a8e2-2e00fd8b602b/episodes/bd38190e-516f-49c0-b47e-6cf663d80986/audio/dc570fd1-7a5e-41e2-b9a4-96deb346c20f/default_tc.mp3">
```

Comment:
we can probably fix the problem of network calls during testing by writing some proxy or middleware code in `web-test-runner.config.js`

[docs](https://modern-web.dev/docs/dev-server/middleware/#rewriting-request-urls)